File: cpp_example/src/test_node.cpp
#include <rclcpp/rclcpp.hpp>

int main(int argc, char **argv)
{
  rclcpp::init(argc, argv);
  auto node = std::make_shared<rclcpp::Node>("carlos");
  RCLCPP_INFO(node->get_logger(), "Test node carlos has been started.");
  rclcpp::spin(node);
  rclcpp::shutdown();
  return 0;
}

File: py_example/py_example/test_node.py
import rclpy
from rclpy.node import Node

class TestNode(Node):
    def __init__(self):
        super().__init__('juanito')
        self.get_logger().info('Test node juanito has been started.')

def main():
    rclpy.init()
    node = TestNode()
    rclpy.spin(node)
    node.destroy_node()
    rclpy.shutdown()

if __name__ == '__main__':
    main()